rviz_cinematographer_camera_poses:
  -
    position:
      x: 0.75
      y: -0.2
      z: 0.65
    orientation:
      x: 0.0
      y: 0.42261843122153303
      z: 0.0
      w: 0.90630770800642
    transition_duration: 1.500000
    wait_duration: 0.000000
  -
    position:
      x: 0.0
      y: 0.55
      z: 0.65
    orientation:
      x: -0.29883645916023044
      y: 0.29883645916023044
      z: 0.6408565418868154
      w: 0.6408565418868154
    transition_duration: 1.500000
    wait_duration: 0.000000
  -
    position:
      x: -0.75
      y: -0.2
      z: 0.65
    orientation:
      x: -0.4226184312214373
      y: 0.0
      z: 0.9063077080062146
      w: 0.0
    transition_duration: 1.500000
    wait_duration: 0.000000
  -
    position:
      x: 0.0
      y: -0.95
      z: 0.65
    orientation:
      x: -0.29883645916023044
      y: -0.29883645916023044
      z: 0.6408565418868154
      w: -0.6408565418868154
    transition_duration: 1.500000
    wait_duration: 0.000000
  -
    position:
      x: 0.75
      y: -0.2
      z: 0.65
    orientation:
      x: 0.0
      y: 0.42261843122153303
      z: 0.0
      w: 0.90630770800642
    transition_duration: 1.500000
    wait_duration: 0.000000
